platform/android/app/src/test/kotlin/coop/polypoly/polypod/polyout/PolyOutTest.kt

```diff
@@ -9,7 +9,6 @@ import coop.polypoly.polypod.features.FeatureStorage
 import coop.polypoly.polypod.polyOut.PolyOut
 import coop.polypoly.polypod.util.FakeAesKeyGenerator
 import coop.polypoly.polypod.util.FakeKeyStore
-import coop.polypoly.polypod.util.MockFeature
```

Comment:
But this means that it's not used anywhere, right? So the code can be removed too.